en-US/win10/SetupTPM.md

```diff
@@ -136,22 +136,22 @@ Firmware TPM (fTPM) requires special Processor/SoC support and whence fTPM is no
 ##<a name="dTPM"></a>Setup dTPM  
 These instructions are applicable for any dTPM module supported on either MBM or RPi2.
 
-1. Get a discrete TPM module and attach it to the MBM.
-2. Based on your discrete TPM module of choice, identify its matching ACPI table [here][3].
-3. Enable testsigning on the device:
-
-        bcdedit /set {current} integrityservices disable
-        bcdedit /set testsigning on
-
-4. Copy the ACPI table to MBM _C:\Windows\System32\ACPITABL.dat_.
-5. Disable fTPM by changing the following UEFI settings:
+1. Get a discrete TPM module and attach it to the MBM/RPi2.
+2. (Applies to MBM) Disable fTPM by changing the following UEFI settings:
 
         Device Manager -> System Setup -> Security Configuration -> PTT = <Disable>
 
-6. Enable dTPM by changing the following UEFI settings:
+3. (Applies to MBM) Enable dTPM by changing the following UEFI settings:
```

Comment:
Can you explain this further? I don't know how to get into UEFI from here; customers also are unlikely to as well.